client/src/modules/payroll/settings/settings.html

```diff
@@ -58,6 +58,45 @@
             on-change-callback="PayrollSettingsCtrl.enableIndexPaymentSetting(value)">
           </bh-yes-no-radios>
 
+          <bh-yes-no-radios
+            label="PAYROLL_SETTINGS.ENABLE_ACTIVATE_PENSION_FUND_BREACKDOWN"
+            value="PayrollSettingsCtrl.enterprise.settings.enable_activate_pension_fund"
+            name="enable_activate_pension_fund"
+            on-change-callback="PayrollSettingsCtrl.enableActivatePensionFundSetting(value)">
+          </bh-yes-no-radios>
+
+          <div class="form-group" ng-class="{ 'has-error' : PayrollSettingsForm.$submitted && PayrollSettingsForm.transaction_type_id.$invalid }">
+            <label class="control-label" translate>
```

Comment:
Would it be possible to just use `<bh-transaction-type-select>`?  If you were able to do this, it would also mean you could remove the `TransactionTypeService` from the controller.

Reply:
I wanted to use the <bh-transaction-type-select> component but this component allows the selection of several transaction types and does not give the possibility of restricting the number of choices